src/app.ts

```diff
@@ -39,50 +40,74 @@ class App {
 			projectHandler.listAllProjects().then((projectList: any) => {
 				res.send(projectList);
 			}).
-			catch((err) => {
-				next(err);
-			});
+				catch((err) => {
+					next(err);
+				});
 		});
 
 		this.express.get('/api/project/getByProjectDid/:projectDid', (req, res, next) => {
 			projectHandler.listProjectByProjectDid(req.params.projectDid).then((projectData: any) => {
 				res.send(projectData);
 			}).
-			catch((err) => {
-				next(err);
-			});
+				catch((err) => {
+					next(err);
+				});
+		});
+
+		this.express.get('/api/project/getProjectAccounts/:projectDid', (req, res, next) => {
+			this.getProjectAccountsFromChain(req.params.projectDid)
+				.then((response: any) => {
+					res.send(response);
+				})
+				.catch((error) => {
+					res.send(error);
+				})
 		});
 
 		this.express.get('/api/did/getByDid/:did', (req, res, next) => {
 			didHandler.getDidDocByDid(req.params.did).then((didDoc: any) => {
 				res.send(didDoc);
 			}).
-			catch((err) => {
-				next(err);
-			});
+				catch((err) => {
+					next(err);
+				});
 		});
 
 		this.express.get('/api/stats/listStats', (req, res, next) => {
 			statsHandler.getStatsInfo().then((stats: any) => {
 				res.send(stats);
 			}).
-			catch((err) => {
-				next(err);
-			});
+				catch((err) => {
+					next(err);
+				});
 		});
 
 		this.express.get('/api/blockchain/:tx', (req, res, next) => {
 			let blockChainConnection = new Connection(this.express.get('chainURL'))
 			blockChainConnection.sendTransaction(req.params.tx).then((result: any) => {
 				res.send(result);
 			}).
-			catch((err) => {
-				next(err);
-			});
+				catch((err) => {
+					next(err);
+				});
 		});
 
 		this.express.use(logger.after);
 	}
+
+	private getProjectAccountsFromChain(projectDid: string) {
```

Comment:
This function should move to project_handler.

Reply:
done